test/Constraints/bridging.swift

```diff
@@ -265,14 +261,12 @@ func rdar19831919() {
 
 // <rdar://problem/19831698> Incorrect 'as' fixits offered for invalid literal expressions
 func rdar19831698() {
-  var v70 = true + 1 // expected-error{{binary operator '+' cannot be applied to operands of type 'Bool' and 'Int'}} expected-note {{expected an argument list of type '(Int, Int)'}}
+  var v70 = true + 1 // expected-error{{cannot convert value of type 'Bool' to expected argument type 'Int'}}
```

Comment:
With some of these cases, it might be worth splitting the line into separate lines so we can ensure that the diagnostic points at the correct operand, e.g., 

```
var v70 = true + 1 // expected-error{{cannot convert value of type 'Bool' to expected argument type 'Int'}}
```

could become

```
var v70 = true // expected-error{{cannot convert value of type 'Bool' to expected argument type 'Int'}}
  + 1 
```

Reply:
Good point!

Reply:
I don't remember if it's documented yet, but the verifier now supports specifying a column number via `expected-error@:42 {{...` which might come in handy here.

Reply:
@owenv Thank you! That would be a much better option, I'll try that out.

Reply:
Done!